requirements.txt

```diff
@@ -4,4 +4,5 @@ graphlib-backport; python_version < '3.9'
 psutil
 pycapnp
 pyzmq
+sortedcontainers
```

Comment:
This does not really add a new dependency, as pycapnp already has it as a dependency.